### PR TITLE
Build/Verify/Test-go/Kubelet jobs for release-1.7

### PR DESCRIPTION
**Pull request by krzyzacy**
/assign @dchen1107 @caesarxuchao

### CODE REVIEW

**Comment by krzyzacy** [comment]
Creating these early before start to create all the e2e jobs.

cc @madhusudancs seems we'll need a new release project for federation 1.7?

File: testgrid/config/config.yaml [hunk 1]
```diff
@@ -2356,3 +2386,28 @@ dashboards:
...
+  - name: test-go-1.7
+    test_group_name: ci-kubernetes-test-go-release-1.7
+  - name: verify-1.7
+    test_group_name: ci-kubernetes-verify-release-1.7
+  - name: kubelet-1.7
+    test_group_name: ci-kubernetes-node-kubelet-1.7
+  - name: federation-build-1.7
+    test_group_name: ci-kubernetes-federation-build-1.7
+
+- name: release-1.7-blocking
```
Comment (caesarxuchao) [inline testgrid/config/config.yaml]: @dchen1107 what are the other tests we need in this list?

**Comment by madhusudancs** [comment]
> cc @madhusudancs seems we'll need a new release project for federation 1.7?

@krzyzacy yeah, we will need one for federation.

File: testgrid/config/config.yaml (hunk 1 above)
Comment (krzyzacy) [inline testgrid/config/config.yaml]: there will be, I'll fill up more next week

**Comment by krzyzacy** [comment]
@madhusudancs let me know whenever you want to run federation jobs against 1.7 branch, create that project and I can help set up from the infra side.

**Pull request merged**